tools/dump_tool/ramdumpParser.py

```diff
@@ -44,12 +44,18 @@
 g_etext_flash=0
 g_stext_ram=0
 g_etext_ram=0
+g_stext_app_micom=0
+g_etext_app_micom=0
+g_stext_app_wifi=0
+g_etext_app_wifi=0
 
 config_path = '../../os/.config'
 elf_path = '../../build/output/bin/tinyara'
 debug_cmd = 'addr2line'
 file_data = 'HeapInfo'
 heapinfo_path = '../../os/include/tinyara/mm/heapinfo_internal.h'
+MICOM_BIN_PATH='../../build/output/bin/micom_dbg'
+WIFI_BIN_PATH='../../build/output/bin/wifi_dbg'
```

Comment:
We could use another name for the application. How about keeping them as default and providing the way to set the path and the name with opt?

Reply:
how about getting names from crash logs itself, if the elf_show_all_bin_addr print name as standard always

Reply:
We should consider the common binary also.